src/router/legacyPermalinkManagement.routerPlugin.js

```diff
@@ -57,7 +57,7 @@ const handleLegacyParams = (legacyParams, store, to, next) => {
             case 'zoom':
                 // if we are using WebMercator as the map projection we need te re-evaluate LV95 zoom,
                 // as it was a zoom level tailor-made for this projection (and not made to cover the whole globe)
-                if (store.state.position.projection.epsg === WEBMERCATOR.epsg) {
+                if (!store.state.position.projection.isSwissProjection) {
                     value = translateSwisstopoPyramidZoomToMercatorZoom(legacyParams[param])
                 } else {
                     value = legacyParams[param]
```

Comment:
Just looking at this diff, it looks wrong ! 
For future it may be wise to have a switch case here and only translate in projection that we support and in default case do a log error.

```javascript
switch(store.state.position.projection.eps) {
  case WEBMERCATOR.epsg:
    value = translateSwisstopoPyramidZoomToMercatorZoom(legacyParams[param])
    break
  case LV95.epsg:
  case LV03.epsg: // not sure this would be correct though ?
    value = legacyParams[param]
    break 
  default:
    log.error(`Unsupported projection`)
    throw Error()
}
```